tutorials/transparent-proxy-and-filtering-on-k8s/index.md

```diff
@@ -0,0 +1,248 @@
+- Deploy the tproxy and the tproxy-podwatch pods using Helm.
+- Deploy example apps to test external access to a Cloud Storage bucket.
+
+## Before you begin
+
+This tutorial assumes you already have a Google Cloud Platform (GCP) account and are familiar with the high level concepts of Kubernetes Pods and Deployments.
+
+## Costs
+
+This tutorial uses billable components of GCP, including:
+
+- [Google Container Engine](https://cloud.google.com/container-engine/pricing){:target="_blank"}
+
+Use the [Pricing Calculator](https://cloud.google.com/products/calculator/#id=f52c2651-4b02-4da3-b8cd-fdbca6ad89a9){:target="_blank"} to estimate the costs for your environment. The resources used in this tutorial are approximately $2.39 per day.
+
+## Checkout the source repository
+
+1. Open [Cloud Shell](https://console.cloud.google.com/cloudshell){:target="_blank"}
+
+2. Clone the repository containing the code for this tutorial:
+
+        git clone https://github.com/danisla/kubernetes-tproxy
+        cd kubernetes-tproxy
+
+    The remainder of this tutorial will be run from the root of the cloned repository directory.
+
+## Create Container Engine cluster and install Helm
+
+1. Create the Container Engine cluster:
+
+        gcloud container clusters create tproxy-example --zone us-central1-f
+
+    This command also automatically configures the kubectl command to use the cluster.
+
+2. Install [Helm](https://github.com/kubernetes/helm) locally in your Cloud Shell instance:
+
+        curl -sL https://storage.googleapis.com/kubernetes-helm/helm-v2.5.1-linux-amd64.tar.gz | tar -xzvf - && sudo mv linux-amd64/helm /usr/local/bin/ && rm -Rf linux-amd64
+
+3. Initialize Helm:
+
+        helm init
+
+    This installs the server side component of Helm, Tiller, in the Kubernetes cluster. The Tiller pod may take a minute to start, run the command below to verify it has been deployed:
+
+        helm version
+
+    You should see the Client and Server versions in the output:
+
+        Client: &version.Version{SemVer:"v2.5.1", GitCommit:"7cf31e8d9a026287041bae077b09165be247ae66", GitTreeState:"clean"}
+        Server: &version.Version{SemVer:"v2.5.1", GitCommit:"7cf31e8d9a026287041bae077b09165be247ae66", GitTreeState:"clean"}
+
+## Install the Helm chart
+
+Before installing the chart, you must first extract the certificates generated by mitmproxy. The generated CA cert is used in the example pods to trust the proxy when making HTTPS requests.
+
+1. Extract the generated certs using Docker:
+
+        cd charts/tproxy
+        docker run --rm -v ${PWD}/certs/:/home/mitmproxy/.mitmproxy mitmproxy/mitmproxy >/dev/null 2>&1
+
+2. Install the chart:
+
+        helm install -n tproxy .
+
+    The output of this command shows you how to augment your deployments to use the init container and trusted certificate configmap volume. Example output below:
+
+        Add the init container spec below to your deployments:
```

Comment:
this renders as code on github make sure it doesn't render as such on website

Reply:
I don't think we can distinguish between code and pre-formatted text in the tutorials. This block is meant to just show the output you see in the console.